### ISSUE TITLE
Add MBARI dive number to FathomNet database imagery

### ISSUE THREAD

**Issue by lauravchrobak** [fathomnet/community-feedback#177]
[Linda's suggestion] Because we are looking at a still frame with no context, it is occasionally helpful to look at the original annotations.

**Comment by hohonuuli** [comment]
What info does she want to look at? I don't think dive number is enough.  Snippet below for fetching info from a user_defined_key.

```sh
curl -X 'POST' \
  'https://m3.shore.mbari.org/anno/v1/query/run' \
  -H 'accept: text/plain' \
  -H 'Content-Type: application/json' \
  -d '{
  "select": [
    "imaged_moment_uuid",
    "observation_uuid",
    "concept",
    "observer",
    "observation_timestamp",
    "video_sequence_name",
    "index_recorded_timestamp",
    "index_elapsed_time_millis",
    "video_uri"
  ],
  "distinct": true,
  "where": [
    {
      "column": "association_uuid",
      "in": [
          "0e9e31f1-65d5-41d9-8a62-da51fb49dd1e",
          "b70a0305-90ee-4784-d261-c1748d3cdd1e"
      ]
    }
  ],
  "strict": true
}'
```

**Comment by lauravchrobak** [comment]
She was envisioning adding the dive number to the details section of the webui so that she could check internally. What do you mean this isn't enough?

**Comment by hohonuuli** [comment]
Some dives have thousands of annotations.

**Comment by kevinsbarnard** [comment]
@hohonuuli Last we chatted about this, you suggested we could add image tags to accomplish this. Maybe something like:
`mbari-dive: Doc Ricketts 0123`

This would be displayed in the image details view as there's already a component that renders the image tags in a list.

**Comment by hohonuuli** [comment]
@kevinsbarnard Yes, adding the dive info is trivial. Just that I'm not sure it will solve the problem Linda has.

**Comment by lauravchrobak** [comment]
@hohonuuli I think linda just wanted to dive number not necessarily the analogous annotation ids. Im not sure i fully understand what problem you think is missing here but if you expand we can ask Linda directly?

**Comment by hohonuuli** [comment]
There's two things: 1. Using the dive id and a concept name to get back to the original annotation seems very cumbersome. 2. Is it OK with the data owners at MBARI if we provide the dive id in FathomNet?
